subworkflows/local/alignment.nf

```diff
@@ -38,13 +38,13 @@ workflow ALIGNMENT {
 
         STAR_ALIGN(FASTP.out.reads, star_index, ch_gtf, false, ch_platform, false)
 
-        SAMTOOLS_INDEX( STAR_ALIGN.out.bam )
+        SAMTOOLS_INDEX( STAR_ALIGN.out.bam_sorted_aligned )
```

Comment:
`.out.bam` seems to work for "real" runs, but crashes for stub runs as it captures multiple output from the stub process